### DISCUSSION TITLE
How to get an error when registering input tag using id

### DISCUSSION THREAD

**Discussion by mhanifmuhsin** [react-hook-form/react-hook-form#7716] [Q&A]
I have a radio button that is dynamically created so that my register uses an id, then i have a problem retrieving validate errors, the value i can't catch. Below is a sample case, I really hope someone can provide a solution, thank you.
sample code :
![code](https://user-images.githubusercontent.com/68141840/152138931-25afcc95-040b-460d-b522-012abe5a825b.png)
<img width="422" alt="Screen Shot 2022-02-02 at 17 45 21" src="https://user-images.githubusercontent.com/68141840/152139282-30662282-996e-4a5c-ae44-8f3b3cbf3404.png">

for catch errors :
![code](https://user-images.githubusercontent.com/68141840/152139519-523cd631-da68-4874-b829-5036a05ba4d3.png)

**Comment by Moshyfawn** [comment]
Have you looked into [`useFieldArray` hook](https://react-hook-form.com/api/usefieldarray)?

**Reply by mhanifmuhsin** [reply]
Okay thanks for the solution I'll try to use it, before I didn't know it existed.
